### DISCUSSION TITLE
PNP Search help for list

### DISCUSSION THREAD

**Discussion by patelgauravpatel** [microsoft-search/pnp-modern-search#2867] [Q&A]
I am trying to create search from a list and not able to see all of my columns in the filter and search results that I want to display.
Need urgent help

**Comment by gretchunkim** [comment]
@patelgauravpatel Did you wait for the SP list to complete index/crawl? Also, visit SP Admin Center to check if the properties/filters are crawled/managed properties.

**Comment by wobba** [comment]
@patelgauravpatel you need to use site columns to have automatic managed properties generated for KQL filtering. If you want them to be in the filter/refiner web part or sort on them, you need to map crawled properties to reusable managed properties. See https://microsoft-search.github.io/pnp-modern-search/scenarios/set-up-managed-properties/

**Reply by patelgauravpatel** [reply]
I have mapped the properties and still not able to pull data or the columns in filter

**Reply by kasperbolarsen** [reply]
Have you tried to fellow the documentation or some of the excellent tutorials such as https://searchexplained.com/pnp-modern-search?
It sounds like some of the basics might need a refresher.
